data/recommendation/recWindow.html

```diff
@@ -4,25 +4,25 @@
     <link rel="stylesheet" type="text/css" href="./style.css">
   </head>
   <body>
-    <div id="welcome" class="welcome-box">
+    <div id="welcome" class="info-box">
       <h1 id="title">Change your password for this site immediately!</h1>
       <p id="date">This site was compromised recently.</p>
       <p id="count">It affected many users, and you may be among them.</p>
       <p>
         If you use your current password for this site on
         any other sites, we recommend you change those as well.
       </p>
-      <div class="buttons">
-        <button id="finished">I've changed my password</button>
-        <button id="no-account">I don't have an account for this site</button>
-        <button id="disable-all">Hide all warnings</button>
+      <div class="button-div">
+        <button id="finished" class="button">I've changed my password</button>
+        <button id="no-account" class="button secondary">I don't have an account</button>
+        <button id="disable-all" class="button secondary">Hide all warnings</button>
```

Comment:
In all of these you're repeating information. If it matters that the div is a div, just say `div.button`. I'd probably go with a classname of `button-group`, and leave off the `div` part. Same with the buttons. Giving a button a classname of button is redundant. A selector like `.button-group > button` would probably be more appropriate.